test/use-swr.test.tsx

```diff
@@ -80,6 +80,18 @@ describe('useSWR', () => {
     )
   })
 
+  it('should not call fetch function when autoLoad is false', async () => {
+    const fetch = jest.fn(() => 'SWR')
+
+    function Page() {
+      const { data } = useSWR('constant-3', fetch)
```

Comment:
```suggestion
      const { data } = useSWR('auto-load', fetch, { autoLoad: false })
```

And for the test, the `autoLoad` option should be disabled (also use a new key here).

Reply:
Ok great, will do